src/test/scala/com/databricks/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -182,10 +182,10 @@ class RedshiftSourceSuite
         |UNLOAD \('SELECT "testbyte", "testbool" FROM
         |  \(select testbyte, testbool
         |    from test_table
-        |    where teststring = \\'Unicode\\'\\'s樂趣\\'\) '\)
+        |    where teststring = \\'\\\\\\\\Unicode\\'\\'s樂趣\\'\) '\)
```

Comment:
Note that in this unit test this line is a regex matching expected output, hence the extra level of backslash escaping.